### PR TITLE
[Admin][Behat] Add shipping order feature

### PR DESCRIPTION
**Pull request by Arminek**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| License         | MIT

### CODE REVIEW

File: features/order/managing_orders/shipping_order.feature [hunk 1]
```diff
@@ -0,0 +1,29 @@
+@managing_orders
+Feature: Shipping an order
+    In order to mark order's shipment as shipped
```
Comment (pjedrzejewski) [inline features/order/managing_orders/shipping_order.feature]: In order to confirm shipping of an order

File: features/order/managing_orders/shipping_order.feature [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    As an Administrator
+    I want to be able to finalize shipment
```
Comment (pjedrzejewski) [inline features/order/managing_orders/shipping_order.feature]: I want to be able to ship a shipment

File: features/order/managing_orders/shipping_order.feature [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        And the customer chose "Free" shipping method with "Cash on Delivery" payment
+        And the customer bought a single "Angel T-Shirt"
+        And I am logged in as an administrator
+
+    Scenario: Finalizing order's shipment
+        Given I view the summary of the order "#00000666"
+        When specify its tracking code as "#00044"
+        And I ship this order
+        Then I should be notified that the order's shipment has been successfully shipped
+        And it shipment state should be shipped
```
Comment (pjedrzejewski) [inline features/order/managing_orders/shipping_order.feature]: its

Comment (pjedrzejewski) [inline features/order/managing_orders/shipping_order.feature]: I'd also wrap the "shipped" in quotes.

**Comment by pjedrzejewski** [comment]
@Arminek Requires rebasing.

**Comment by pjedrzejewski** [comment]
Thank you Arek!

File: src/Sylius/Bundle/AdminBundle/Resources/config/routing/partial/order.yml [hunk 4]
```diff
@@ -7,3 +7,27 @@ sylius_admin_partial_order_latest:
...
+    path: /orders/{id}/shipments/{shipmentId}/ship
+    methods: [GET, PUT]
+    defaults:
+        _controller: sylius.controller.shipment:updateAction
+        _sylius:
+            repository:
+                method: findByOrderIdAndId
+                arguments:
+                    orderId: $id
+                    id: $shipmentId
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/AdminBundle/Resources/config/routing/partial/order.yml]: Actually I think the routing should be `/orders/{orderId}/shipments/{id}/ship` we have this convention for promotion coupons, variants etc. and the then arguments would be `orderId: $orderId, id: $id`.

This should be also updated for the payments.

Comment (pjedrzejewski) [inline src/Sylius/Bundle/AdminBundle/Resources/config/routing/partial/order.yml]: Totally agree, @Arminek can you drop a quick PR to fix that?

**Pull request merged**